quickstart/configure-mayastor.md

```diff
@@ -1,45 +1,45 @@
 # Configure Mayastor
 
-## Create Mayastor Pool\(s\)
+## Create DiskPool\(s\)
 
 
-### What is a Mayastor Pool?
+### What is a DiskPool?
 
-When a Mayastor node allocates storage capacity for a replica of a Persistent Volume it does so from a Mayastor Pool. Each Mayastor node may create and manage zero, one, or more such pools. The ownership of a pool by a node is exclusive. A pool can manage only one block device, which constitutes the entire data persistence layer for that pool and thus defines its maximum storage capacity.
+When a node allocates storage capacity for a replica of a persistent volume (PV) it does so from a DiskPool. Each node may create and manage zero, one, or more such pools. The ownership of a pool by a node is exclusive. A pool can manage only one block device, which constitutes the entire data persistence layer for that pool and thus defines its maximum storage capacity.
 
 {% hint style="info" %}
-Note that in the current version of Mayastor pools support only thick provisioning.
+The current version of DiskPool only supports thick provisioning.
```

Comment:
I think that should be _this_ version of Mayastor.  Since this branch refers to version 2.0.x only and so that fact will always be true.  What the current version (which may be anything after this release) does is beyond definitive claim in this version of the doc.

Reply:
Updated the text.